### ISSUE TITLE
architecture shows bad results

### ISSUE THREAD

**Issue by e0xextazy** [keonlee9420/StyleSpeech#4]
Hi, i have completely repeated your steps for learning. During training, style speech loss fell down, but after learning began, meta style speech loss began to grow up. Can you help with training the model? I can describe my steps in more detail.

**Comment by keonlee9420** [comment]
Hi @e0xextazy, thanks for the report. I found that the current implementation updates both generator and discriminator when the discriminator is updated by `ScheduledOptimDisc`, but it should be the discriminator only. I fixed the code and pushed to `main` branch, and my training log shows the expected results as follows. (grey: previous version, blue: fixed version)
![image](https://user-images.githubusercontent.com/34522972/132279800-71c28cce-97ba-4c44-83d3-f9f2940d56ef.png)
The pre-trained model is just updated as well.
